tests/Models/TeamTest.php

```diff
@@ -30,6 +31,7 @@ public function testDelete(): void
         $team->members()->create(['user_id' => User::factory()->create()->getKey()]);
         Chat\Message::factory()->create(['channel_id' => $team->channel, 'user_id' => $team->leader_id]);
         $team->applications()->create(['user_id' => User::factory()->create()->getKey()]);
+        $team->statistics()->create(['ruleset_id' => 0]);
```

Comment:
other team in the test should also have `statistics` (and probably add `applications` as well)